numpy/lib/function_base.py

```diff
@@ -1332,9 +1332,13 @@ def interp(x, xp, fp, left=None, right=None, period=None):
 
     Notes
     -----
-    Does not check that the x-coordinate sequence `xp` is increasing.
-    If `xp` is not increasing, the results are nonsense.
-    A simple check for increasing is::
+    The x-coordinate sequence is expected to be increasing, but this is not
+    explicitly enforced.  However, if the sequence `xp` is non-increasing,
+    interpolation results are meaningless.
+
+    Note that, since `NaN` is unsortable, `xp` then also cannot contain `NaN`s.
```

Comment:
```suggestion
    Note that, since NaN is unsortable, `xp` also cannot contain NaNs.
```
I think the single backticks are not quite correct and `` `NaN`s `` probably would not be rendered correctly/could be an error (CircleCI is currently not running, so not sure.)

Reply:
Let's try that.

Reply:
Thanks! Was probably OK to begin with, but... Will just put it in.